index.js

```diff
@@ -1,14 +1,7 @@
-/*
-play this: https://www.youtube.com/watch?v=d-diB65scQU
+const server = require("./server");
 
-Sing along:
+const port = process.env.PORT ? process.env.PORT : 4000;
```

Comment:
You could also write it `const port = process.env.PORT || 4000;` This will check to see if there's an environment variable provided, if not it will fall back to the secondary value.